examples/hello.nix

```diff
@@ -8,6 +8,11 @@ let
   # <nixpkgs> is set to the value designated by the nixpkgs input of the
   # jobset configuration.
   pkgs = (import <nixpkgs> {});
-in {
-  hello = pkgs.hello;
+in
+{
+  hello = pkgs.hello.overrideAttrs (
+    { ... }: {
+      never_hit_cache = toString builtins.currentTime;
```

Comment:
Hm, we probably shouldn't use undocumented impure functions in examples...

Reply:
Good point. I removed that commit.